core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1412,6 +1412,15 @@ public enum Property {
           + "constraint.",
       "2.0.0"),
 
+  TABLE_ENABLE_ERASURE_CODES("table.file.ec.enable", "false", PropertyType.BOOLEAN,
```

Comment:
I think TABLE_FILE_REPLICATION and TABLE_ENABLE_ERASURE_CODES should be mutually exclusive.